src/Sylius/Bundle/CoreBundle/Form/EventSubscriber/AddAuthorGuestTypeFormSubscriber.php

```diff
@@ -40,11 +40,11 @@ public function preSetData(FormEvent $event)
         $form = $event->getForm();
         /** @var ReviewInterface $review */
         $review = $event->getData();
-        $author = $form->getConfig()->getOption('author');
+        $currentLoggedAuthor = $form->getConfig()->getOption('author');
```

Comment:
Btw. can't we set the author on review just before passing it to form? It will make that logic a lot simplier.